tauri/src/alchemy/mod.rs

```diff
@@ -77,6 +80,7 @@ impl Alchemy {
             let res = (res["result"]).clone();
             let res: AlchemyResponse = serde_json::from_value(res)?;
             self.db.save_balances(res, chain_id).await?;
+            self.window_snd.send(Notify::TxsUpdated.into())?;
```

Comment:
Actually for this we should add a new `BalancesUpdated` instead. we'll also eventually monitor transactions via alchemy, but that's a separate request

Reply:
in the block_listener we're calling that as well.
![image](https://github.com/iron-wallet/iron/assets/97689347/1bbe2e4b-60d8-4e6a-9a8a-c5a1d571b6f7)
What's the best way here? let the block_listener keep on only calling the TxsUpdated and listen both events on the frontend balancesProvider or make the balances provider only listening to BalancesUpdated and add two `self.window_snd.send` (TxsUpdated and BalancesUpdated) on the block_listener?

Reply:
I think the second (send both events)